CIP-0035/README.md

```diff
@@ -156,26 +181,22 @@ So, for example, a single CIP could propose multiple new builtin functions, or a
 
 Proposals for additions to the set of Plutus Core builtins SHOULD be proposed in a CIP and SHOULD adhere to the following additional process.
 
-In order to move to Draft status, it MUST include:
+In order to move to Proposed status, it MUST include:
 - In the Specification:
     - Names and types/kinds for the new functions or types.
     - A source for the implementation (e.g. a library which can be linked against); or a generic description of the functionality which is implementable in any programming language.
-    - In the Rationale:
+    - For new types: a precise description of the measure used for the size of a value of that type.
+    - For new builtin functions: a costing function for the builtin function.
+- In the Rationale:
     - An argument for the utility of the new builtins.
+    - If an external implementation is provided: an argument that it is trustworthy.
```

Comment:
There are certain extra conditions that we'll require to hold: for example the builtin must always terminate and must never throw any exceptions, and its behaviour should be predictable (there shouldn't be any worst-case behaviour that's significantly worse than the costing function would predict).  I don't think we have a formal list of such things yet, but we'll need one eventually and it's not clear to me whether it should go in this CIP or somewhere else.  Maybe we need a builtin implementer's guide.

Reply:
I'll add some of that.